WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -2464,18 +2382,69 @@ local function addCollapsibleHeader(options, key, title, order, isGroupTab)
     image = function()
       local isCollapsed = WeakAuras.IsCollapsed("collapse", "region", key, false)
       return isCollapsed and "Interface\\AddOns\\WeakAuras\\Media\\Textures\\expand" or "Interface\\AddOns\\WeakAuras\\Media\\Textures\\collapse", 18, 18
-    end
+    end,
+    hidden = hiddenFunc
   }
 
   options[key] = {
     type = "description",
     name = title,
     order = order + 0.2,
-    width = WeakAuras.doubleWidth - 0.15,
-    fontSize = "large"
+    width = titleWidth,
+    fontSize = "large",
+    hidden = hiddenFunc
   }
-  return function()
-    return WeakAuras.IsCollapsed("collapse", "region", key, false)
+
+  if hasUp then
+    options[key .. "upButton"] = {
+      type = "execute",
+      name = "",
+      order = order + 0.3,
+      width = 0.15,
+      func = input.__up,
+      image = "Interface\\AddOns\\WeakAuras\\Media\\Textures\\moveup",
```

Comment:
It occurs to me that we could might do well to make this less magic in the same way that we did for new feature indicators

Reply:
I'm not yet bothered enough to do that :)